Notebooks/DC2Cookbook.ipynb

```diff
@@ -393,19 +393,35 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Make the `TemplateCoadd`s\n",
+    "### Make the `SkyMap` [issue [#121](https://github.com/LSSTDESC/DC2_Repo/issues/121)]\n",
+    "\n",
+    "The first thing we need to make is the sky map on which the coadded images will be projected for all bands. In order to ensure that the sky map will not depend on the list of input visits (their location on the sky or on the time and place the reprocessing happens), the sky map should be absolute (using `makeSkyMap.py` and the appropriate configuration) instead of relative to an input field (as done with `makeDiscreteSkyMap.py`). Making such a sky map will create thousands of tracts and patches, that will be fixed on the sky and usable in different reprocessing sessions. \n",
+    "\n",
+    "One possibility could be to use the [Rings sky map](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/ringsSkyMap.py#L42), as done for the HSC data reprocessing with the LSST stack (and currently for CFHT data reprocessing as well). In order to make this sky map, some modifications of the `lsstSimMapper.py` [script](https://github.com/lsst/obs_lsstSim/blob/master/python/lsst/obs/lsstSim/lsstSimMapper.py) must be done to match what has been done in `hasMapper.py` ([here](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/ringsSkyMap.py#L42) and [here](https://github.com/lsst/obs_subaru/blob/master/python/lsst/obs/hsc/hscMapper.py#L286)). A configuration file (`makeSkyMapConfig.py` below) is then needed to run `makeSkyMap.py`, containing information on how to parametrize the rings map. An example of such a configuration can be taken from the HSC reprocessing [repository](https://github.com/LSSTDESC/ReprocessingTaskForce/blob/master/config/w_2017_49/cfht/makeSkyMapConfig.py), which contains:\n",
+    "\n",
+    "```\n",
+    "config.skyMap.name = \"rings\"\n",
+    "config.skyMap[\"rings\"].numRings = 120\n",
+    "config.skyMap[\"rings\"].projection = \"TAN\"\n",
+    "config.skyMap[\"rings\"].tractOverlap = 1.0/60 # Overlap between tracts (degrees)\n",
+    "config.skyMap[\"rings\"].pixelScale = 0.185\n",
+    "```\n",
```

Comment:
@jchiang87 : in his new `makeSkyMap` cookbook recipe section, @nicolaschotard is suggesting we adapt an LSST script, as HSC have done, and provide a configuration, analogous to the CFHTLS repro one. Can you suggest a sentence of the form "this modified script and configuration should live in the DESC repo XXX" - I'm not sure what we should be forking. Nicolas, feel free to suggest a sensible scheme, too.

Reply:
The configuration will follow the choice of sky map we will use. What will have to be updated accordingly to this choice is the `lsstSimMapper.py` script that leaves in the `obs_lsstSim` github repository ([here](https://github.com/lsst/obs_lsstSim/blob/master/python/lsst/obs/lsstSim/lsstSimMapper.py)). All changes should actually be done in `obs_lsstSim`, and the configuration file should also be put in there as a default configuration for the sky map. But again, DM people might have a different view on that.

Reply:
I opened an (Jira ticket)[https://jira.lsstcorp.org/browse/DM-13377] about that ().